Makefile

```diff
@@ -479,8 +479,51 @@ rpm:
 
 .PHONY: docker-image
 docker-image: static
-	cp $$GOPATH/bin/skydive contrib/docker/
-	docker build -t ${DOCKER_IMAGE}:${DOCKER_TAG} -f contrib/docker/Dockerfile contrib/docker/
+	cp $$GOPATH/bin/skydive contrib/docker/skydive.$$(uname -m)
+	cp /bin/true contrib/docker/qemu-$$(uname -m)-static
```

Comment:
why we should generate a fake qemu-%{arch}-static binary ?

Reply:
Its because the `contrib/docker/Dockerfile` is used for native and multiarch and its `COPY`line can't be optional (build ARG could be used but only as a filename) or tolerate failure. I was avoiding splitting/duplicating the Dockerfile for this difference. And empty file is probably sufficient. I'll change to that.